### DISCUSSION TITLE
アイドル(IDOL)　Update

### DISCUSSION THREAD

**Discussion by mirai-mamori** [mirai-mamori/Sakurairo#688] [General]
![banner-ver2 6](https://user-images.githubusercontent.com/61381142/232706154-c3311c3c-8fbb-4cbb-b1c9-76d2124995c1.png)

![cn-ver2 6](https://user-images.githubusercontent.com/61381142/232706061-888283a7-7614-4b60-a08e-3b6ced695f27.png)

![cn-ver2 6info](https://user-images.githubusercontent.com/61381142/232706208-7353138a-4f57-4358-be18-ae1b707fe483.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/mirai-mamori/Sakurairo/releases/tag/2.6.0'>アイドル(IDOL)　Update </a>.</em>

**Comment by KotoriK** [comment]
瞳宝，坏！

**Reply by mirai-mamori** [reply]
> 瞳宝，坏！

K宝，好！
